src/elfpy/simulators.py

```diff
@@ -329,53 +331,57 @@ def run_simulation(self, override_dict=None):
                         last_user_action_time = self.market.time
                         # if self.verbose:
-                            # print(
-                            #     "YieldSimulator.run_simulation:"
-                            #     f"\n\ttime = {self.market.time}"
-                            #     f"\n\tuser trade = {user_action}"
-                            #     f"\n\tuser_wallet = {user.wallet}"
-                            #     f"\n\tinit pool apy = {self.init_pool_apy}"
-                            #     f"\n\ttrades = {self.market.base_asset_orders + self.market.token_asset_orders}"
-                            #     f"\n\tinit_share_price = {self.market.init_share_price}"
-                            #     f"\n\tshare_price = {self.market.share_price}"
-                            #     f"\n\tamount = {self.trade_amount}"
-                            #     f"\n\treserves = {(self.market.share_reserves, self.market.bond_reserves)}"
-                            # )
+                        # print(
+                        #     "YieldSimulator.run_simulation:"
+                        #     f"\n\ttime = {self.market.time}"
+                        #     f"\n\tuser trade = {user_action}"
+                        #     f"\n\tuser_wallet = {user.wallet}"
+                        #     f"\n\tinit pool apy = {self.init_pool_apy}"
+                        #     f"\n\ttrades = {self.market.base_asset_orders + self.market.token_asset_orders}"
+                        #     f"\n\tinit_share_price = {self.market.init_share_price}"
+                        #     f"\n\tshare_price = {self.market.share_price}"
+                        #     f"\n\tamount = {self.trade_amount}"
+                        #     f"\n\treserves = {(self.market.share_reserves, self.market.bond_reserves)}"
+                        # )
                 # TODO: convert to proper logging
-                if self.verbose and (self.market.time - last_user_action_time > 0.1) and (self.market.time - last_user_action_time) % 0.5 < 1/365/self.num_blocks_per_day/2:
+                if (
+                    self.verbose
+                    and (self.market.time - last_user_action_time > 0.1)
+                    and (self.market.time - last_user_action_time) % 0.5 < 1 / 365 / self.num_blocks_per_day / 2
+                ):
                     print(
-                        f"t={bcolors.HEADER}{self.market.time}{bcolors.ENDC}"+
-                        f" reserves=[x={bcolors.OKBLUE}{self.market.share_reserves}{bcolors.ENDC}"+
-                        f",y={bcolors.OKBLUE}{self.market.bond_reserves}{bcolors.ENDC}]\n"+
-                        f" no user action 😴"+
-                        f" user report = {self.user_list[0].status_report()}"
+                        f"t={bcolors.HEADER}{self.market.time}{bcolors.ENDC}"
+                        + f" reserves=[x={bcolors.OKBLUE}{self.market.share_reserves}{bcolors.ENDC}"
```

Comment:
I don't think you need the plus at all

Reply:
This was automatically done by my auto-formatter. I'll see if I can add something about this to the black configuration.